src/components/menus/TopMenu.module.scss

```diff
@@ -212,7 +212,7 @@
 }
 
 .locationBtn:focus {
-  outline: 2px solid #d3005b; /* Example focus outline */
+  outline: 2px solid black;
   outline-offset: 2px;
 }
 
```

Comment:
I've tried playing around with this to get it to not select the | separator but can't get it. Tried taking it off the button itself and just directly on the span etc, setting the pseudo element to no focus etc. Not entirely sure how to get it so open to any ideas!

Reply:
This did the trick for me:

```
.locationBtn:focus {
  span {
    outline: 2px solid black;
    outline-offset: 2px;
  }
}
```